README.md

```diff
@@ -281,7 +281,7 @@ This is the same app used by `serve` and `export` and initializes the cache,
 using the cached styles when available.
 
 ```python
-create_app(path=None, gfm=False, context=None, username=None, password=None, render_offline=False, render_wide=False, render_inline=False, text=None)
+create_app(path=None, gfm=False, context=None, username=None, password=None, api_url=None, render_offline=False, render_wide=False, render_inline=False, text=None)
```

Comment:
Can you move the new argument to the last position, after `text`?

This is it's backwards compatible with any libraries using grip with positional arguments.